specs/market_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './spec_helper'
+require_relative '../far_mar'
+require_relative '../lib/farmar_market'
```

Comment:
You technically don't need this line. `../far_mar` is requiring all of the data classes, making the `require` here redundant.